docs/checkers/index.md

```diff
@@ -891,6 +891,20 @@ Example File 2
 
 ```
 
+### Consider using from import (R0402) [](#R0402)
+
+Instead of aliasing a multi-level import's final level as itself, consider using a from import.
```

Comment:
@ckatsube can you try rewording this to make it more beginner-friendly? It's okay to have a description that is longer and/or more concrete, which is easier to understand for beginners. In particular, terminology like "aliasing" and "multi-level import" are tricky to understand.